cmd/zed/zed.d/zed-functions.sh

```diff
@@ -126,7 +126,7 @@ zed_lock()
 
     # Obtain a lock on the file bound to the given file descriptor.
     #
-    eval "exec ${fd}> '${lockfile}'"
```

Comment:
Just so I understand correctly, you use the append (`>>`) instead of the truncate (`>`) is to keep us from needlessly recreating the lockfile if it already exists?  Is that right?

Reply:
In so many words, yeah. It allows us to lock on any file (that we have write permissions for) instead of only dedicated lock files, since the locking process itself no longer alters the file in any way.

Reply:
Ok, could you add that to the commit message, or add it as a comment to the code, just so future generations know what's happening:
```
zed-functions.sh: zed_lock(): don't truncate lock

Use append (>>) instead of truncate (>) to let us lock on any file
(that we have write permissions for) instead of only dedicated lock
files, since the locking process itself no longer alters the file in
any way.
```

Reply:
Sure, updated